### DISCUSSION TITLE
Deploying first contract and getting a testnet on chainlist

### DISCUSSION THREAD

**Discussion by sidrae001** [Cyfrin/foundry-full-course-cu#2785] [General]
I have two problems I want to discuss.
I'm currently learning how to deploy my first contract. On deploy and run transactions, when I select 'inject provider - metamask' my wallet shows '0 ether' but i've got about 0.3 sepolia testnet which should reflect on my account after selecting 'inject provider - metamask' and i'm also supposed to see the sepolia faucet beneath when I select the inject provider but what I see is custom (300) network. 
Also the second problem is on chainlist, when I choose the faucet I would like to receive say 'sepolia testnet' I try to click on 'add to metamask' but it refuses to load or go through. What can I do to solve these little problems because they have put my learning to a temporal hold.

**Comment by EngrPips** [comment]
Please use `wallet-connect` option instead of `inject provider` and maybe refresh your browser and retry the chainlist website.
